src/main/kotlin/graphics/scenery/volumes/HasTransferFunction.kt

```diff
@@ -11,4 +16,43 @@ interface HasTransferFunction {
     var minDisplayRange : Float
     var maxDisplayRange : Float
     var range: Pair<Float, Float>
+
+    /**
+     * Load transfer function and display range from file that was written by [HasTransferFunction.toFile]
+     */
+    fun loadTFFromFile(file: File){
+        val tf = TransferFunction()
+        val inputStream: InputStream = file.inputStream()
+        var isRangeSet = false
+        inputStream.bufferedReader().forEachLine {
+            val line = it.trim().split(";").mapNotNull(kotlin.String::toFloatOrNull)
+            if (line.size == 2){
+                if (!isRangeSet){
+                    this.minDisplayRange = line[0]
+                    this.maxDisplayRange = line[1]
+                    isRangeSet = true
+                } else {
+                    tf.addControlPoint(line[0], line[1])
+                }
+            }
+        }
+        this.transferFunction = tf
+    }
+
+    /**
+     * Write transfer function to file in a human-readable way.
+     * Format:
+     * First line is the display range sepaerated by a semicolon.
+     * All following lines are tf control points.
+     */
+    fun toFile(file: File){
```

Comment:
This should rather be called `saveTransferFunctionToFile()`, otherwise this might indicate serialisation of the entire object